packages/node/src/breadcrumbs/FileBreadcrumbsStorage.ts

```diff
@@ -27,12 +28,16 @@ export class FileBreadcrumbsStorage implements BreadcrumbsStorage {
     constructor(
         private readonly _mainFile: string,
         private readonly _fallbackFile: string,
+        fileSystem: NodeFileSystem,
         maximumBreadcrumbs: number,
     ) {
-        this._writer = new AlternatingFileWriter(_mainFile, _fallbackFile, maximumBreadcrumbs);
+        this._writer = new AlternatingFileWriter(_mainFile, _fallbackFile, maximumBreadcrumbs, fileSystem);
```

Comment:
It is not part of this pull request, however, can we adjust the code also here? The number of breadcrumbs shouldn't extend the desired number of breadcrumbs declared by the user
```suggestion
        this._writer = new AlternatingFileWriter(
            _mainFile,
            _fallbackFile,
            Math.floor(maximumBreadcrumbs / 2),
            _fileSystem,
        );
```

Reply:
Changed.